### PR TITLE
Faster JobCallbackRequestMakerTest

### PR DESCRIPTION
**Pull request by juhoautio**
This test used to take ~5s. Now it's ~0.3s.

Remaining slowest test is `azkaban.execapp.JobRunnerTest`: 2.513s.

All enabled tests now run in ~5s \o/

### CODE REVIEW

**Comment by juhoautio** [comment]
@HappyRay, could you check this please?

File: azkaban-exec-server/src/main/java/azkaban/execapp/event/JobCallbackRequestMaker.java [hunk 1]
```diff
@@ -84,18 +90,14 @@ public static JobCallbackRequestMaker getInstance() {
   private JobCallbackRequestMaker(Props props) {
 
     int connectionRequestTimeout =
-        props.getInt("jobcallback.connection.request.timeout",
-            DEFAULT_TIME_OUT_MS);
+        props.getInt(JOBCALLBACK_CONNECTION_REQUEST_TIMEOUT, DEFAULT_TIME_OUT_MS);
```
Comment (HappyRay) [inline azkaban-exec-server/src/main/java/azkaban/execapp/event/JobCallbackRequestMaker.java]: Thank you for making them constant and moving them to the central location.

**Comment by HappyRay** [comment]
Excellent. Thanks!

LGTM. Could you update the branch or do you mind that I update the branch so that we can merge?

It appears that you shortened the various Jetty timeout values. It would be great if you can describe what you did to speed up the tests.

**Comment by juhoautio** [comment]
Updated branch.

How this was speeded up: This test was being run using the default timeout values. In the test the normal request time is so short that we can set lower timeouts in test props and still test the timeout scenarios reliably.

**Comment by HappyRay** [comment]
Thanks!

**Pull request merged**